src/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -62,6 +88,7 @@ public class HttpRequestData
         public string Method;
         public string Path;
         public List<HttpHeaderData> Headers { get; }
+        public IntPtr Handle;
```

Comment:
Could you add some comment what's this Handle? (or rename it) Does not seem immediately obvious